### PR TITLE
Dont try to update web profile on readonly wallets

### PR DESCRIPTION
**Pull request by brunobar79**
Fixes RNBW-1483

### CODE REVIEW

**Review by nicoglennon — APPROVED** [review]
🦾

File: src/hooks/useWebData.js [hunk 1]
```diff
@@ -81,6 +83,9 @@ export default function useWebData() {
 
   const updateWebProfile = useCallback(
     async (address, name, color) => {
+      const wallet = findWalletWithAccount(address);
+      if (wallet.type === WalletTypes.readOnly) return;
+
       if (!webDataEnabled) return;
```
Comment (skylarbarrera) [inline src/hooks/useWebData.js:84]: I feel like we could combine both returns

**Review by skylarbarrera — APPROVED** [review]
🤝

File: src/hooks/useWebData.js (hunk 1 above)
Comment (brunobar79) [reply src/hooks/useWebData.js:84]: yes and no 😅 

I'm gonna move the first return to the top, so we can exit early and maybe prevent the findWalletWithAccount lookup 

🤝

**Pull request merged**